paddle/framework/framework.proto

```diff
@@ -118,6 +118,14 @@ message LoDTensorArrayDesc {
 
 message ReaderDesc { repeated LoDTensorDesc lod_tensor = 1; }
 
+message ChanEleDesc { repeated VarDesc meta_data = 1; }
+
+message ChanDesc {
+  repeated ChanEleDesc channel_type = 1;
+  required int32 cap = 2 [ default = 0 ];
+  required string name = 3;
```

Comment:
I don't understand why would we need `name` here.  I can see that we do need `cap`.

A reference design is the Go's channel definition: https://github.com/golang/go/blob/4fc9565ffce91c4299903f7c17a275f0786734a1/src/runtime/chan.go#L17-L29

We are particularly interested in `elemtype`.